src_c/line.c

```diff
@@ -234,7 +234,8 @@ pg_line_copy(pgLineObject *self, PyObject *_null)
 }
 
 static PyObject *
-pg_line_is_parallel(pgLineObject *self, PyObject *const *args, Py_ssize_t nargs)
+pg_line_is_parallel(pgLineObject *self, PyObject *const *args,
```

Comment:
Was this changed via a linter? This change should be reverted.

Reply:
Yes this was my IDE automatic clang formatter